app/controllers/register/establishers/EstablisherKindController.scala

```diff
@@ -43,40 +43,34 @@ class EstablisherKindController @Inject()(
                                            getData: DataRetrievalAction,
                                            requireData: DataRequiredAction,
                                            formProvider: EstablisherKindFormProvider
-                                         ) (implicit val ec: ExecutionContext) extends FrontendController with Retrievals with I18nSupport with Enumerable.Implicits {
+                                         )(implicit val ec: ExecutionContext) extends FrontendController with Retrievals with I18nSupport with Enumerable.Implicits {
 
   private val form = formProvider()
 
   def onPageLoad(mode: Mode, index: Index): Action[AnyContent] = (authenticate andThen getData andThen requireData).async {
     implicit request =>
-      retrieveSchemeName {
-        schemeName =>
-          val redirectResult = request.userAnswers.get(EstablisherKindId(index)) match {
-            case None => Ok(establisherKind(appConfig, form, mode, index))
-            case Some(value) => Ok(establisherKind(appConfig, form.fill(value), mode, index))
-          }
-          Future.successful(redirectResult)
+      val redirectResult = request.userAnswers.get(EstablisherKindId(index)) match {
```

Comment:
Could combine into two lines and eliminate code duplication thus making it more readable:-
Future.successful(Ok(establisherKind(appConfig,
   request.userAnswers.get(EstablisherKindId(index)).fold(form)(form.fill), mode, index)))